src/iconsManifest/supportedExtensions.ts

```diff
@@ -2642,6 +2642,11 @@ export const extensions: IFileCollection = {
       extensions: ['juice'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'cmrc',
+      extensions: ['cmrc'],
```

Comment:
Currently the icon is applied to the `.cmrc` file extension. The following change will apply the icon to the `.cmrc` filename instead.

```suggestion
      extensions: ['.cmrc'],
      filename: true,
```

Also ideally this list should be sorted alphabetically, but IMO that’s not blocking.